deployment/openshift/webapi_bc.yaml

```diff
@@ -46,7 +46,7 @@ objects:
     strategy:
       type: Docker
       dockerStrategy:
-        dockerfilePath: Openshift_Dockerfile
+        dockerfilePath: Dockerfile
```

Comment:
I had to create separate docker files for openshift and aot builds since they have different sources where they pull the image from..
In the lab , we use artifcatory.. but in aot build , we use just docker hub..
Is it okay if we keep Openshift_Dockerfile reference in the bc?